build_tools/cmake/ctest_all.sh

```diff
@@ -168,7 +168,10 @@ if (( ${#excluded_tests[@]} )); then
   ctest_args+=("--exclude-regex ${excluded_tests_regex}")
 fi
 
-echo "*************** Running CTest ***************"
-
 set -x
+
+echo "*************** Running CTest ***************"
 ctest ${ctest_args[@]}
+
+echo "**************** llvm-external-projects tests ***************"
```

Comment:
We also don't want this in runtime builds like `build_test_runtime`. Might need a new environment variable at the top of the file to control whether this is included (ew...).